inc/Remote/ApiCore.php

```diff
@@ -618,18 +618,10 @@ public function createUser($userStruct)
         $mail = trim(preg_replace('/[\x00-\x1f:<>&%,;]+/', '', $userStruct['mail'] ?? ''));
         $groups = $userStruct['groups'] ?? [];
 
-        $notify = (boolean)$userStruct['notify'] ?? false;
+        $notify = (bool)$userStruct['notify'] ?? false;
 
-        if (empty($user)) {
-            throw new RemoteException('user not supplied', 114);
-        }
-
-        if (empty($name)) {
-            throw new RemoteException('name not supplied', 114);
-        }
-
-        if (!mail_isvalid($mail)) {
-            throw new RemoteException('mail not valid', 114);
+        if ($user === '' || $name === '' || !mail_isvalid($mail)) {
+            return false;
```

Comment:
Detailed error messages are not working? Of why change to Boolean?

Reply:
Or is this related to the server errors? 
These are fixed with https://github.com/splitbrain/dokuwiki/pull/3620?

Reply:
I believe exceptions should work now and would be preferable. @satwareAG-ironMike can you have a look at this so we can finally merge it?